log-manager/src/main/java/io/airlift/log/StaticFormatter.java

```diff
@@ -62,8 +79,16 @@ public String format(LogRecord record)
                 .append('\t')
                 .append(Thread.currentThread().getName())
                 .append('\t')
-                .append(record.getLoggerName())
-                .append('\t')
+                .append(record.getLoggerName());
+
+        if (!logAnnotations.isEmpty()) {
+            stringWriter.append('\t')
```

Comment:
This can be
```java
stringWriter.append('\t');
Joiner.on(",").withKeyValueSeparator("=").appendTo(stringWriter, logAnnotations);
```

Reply:
.appendTo throws IOException, so I just did .join and appended the output